### DISCUSSION TITLE
ElectronBot组装视频

### DISCUSSION THREAD

**Discussion by GreenShadeZhang** [peng-zhihui/ElectronBot#129] [General]
[ElectronBot组装视频](https://www.bilibili.com/video/BV1oY4y1v7oN?share_source=copy_web)
我录了一个整体的组装视频，有没有人分享下舵机的驱动板下载固件的那几个引脚的焊接图或者你们是怎么连接下载器的，然后分享下，那三个引脚好小，我都不知道是焊上去还是用针顶着好一些了。

**Comment by peng-zhihui** [comment]
用SH1.0的3P接插件顶着下载就行（可能需要有个人配合一下）

**Reply by GreenShadeZhang** [reply]
好的 谢谢 我到时候试试

**Comment by peng-zhihui** [comment]
非常感谢出视频，我帮你加到项目描述里~

**Reply by GreenShadeZhang** [reply]
能帮助大家 我也很开心

**Comment by wenFrankieeee** [comment]
固定螺丝是用什么规格的？
